.swiftlint.yml

```diff
@@ -0,0 +1,74 @@
+included:
+  - Sources
+
+disabled_rules:
+  - trailing_whitespace
+  - trailing_comma
+  - void_function_in_ternary
+#  - force_cast
+
+opt_in_rules:
```

Comment:
A lot of the rules here are not necessary for this package - I've copy-pasta'd them from another project where I had a _stricter_ linting setup 🧑‍⚖️